packages/dev/core/src/Rendering/GaussianSplatting/gaussianSplatting.ts

```diff
@@ -0,0 +1,342 @@
+import { Effect } from "../../Materials/effect";
+import { ShaderMaterial } from "../../Materials/shaderMaterial";
+import { Matrix, Quaternion, TmpVectors, Vector2 } from "../../Maths/math.vector";
+import { Mesh } from "../../Meshes/mesh";
+import { VertexData } from "../../Meshes/mesh.vertexData";
+import type { Observer } from "../../Misc/observable";
+import { Tools } from "../../Misc/tools";
+import type { Scene } from "../../scene";
+import type { Nullable } from "../../types";
+
+/**
+ *
+ */
+export class GaussianSplatting {
+    private _vertexCount: number = 0;
+    private _positions: Float32Array;
+    private _uBuffer: Uint8Array;
+    private _covA: Float32Array;
+    private _covB: Float32Array;
+    private _sceneDisposeObserver: Nullable<Observer<Scene>>;
+    private _sceneBeforeRenderObserver: Nullable<Observer<Scene>>;
+    private _material: ShaderMaterial;
+    /**
+     *

```

Comment:
Just flagging the empty comments as we discussed

Reply:
fixed